api/NodeIterator.json

```diff
@@ -0,0 +1,488 @@
+{
+  "api": {
+    "NodeIterator": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/NodeIterator",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "3.5"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": "9"
+          },
+          "opera_android": {
+            "version_added": "9"
+          },
+          "safari": {
+            "version_added": "3"
+          },
+          "safari_ios": {
+            "version_added": "3"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "baseURI": {
```

Comment:
The key needs to be `expandEntityReferences` here

Reply:
OK - good spot - done!